Nimble.podspec

```diff
@@ -51,5 +51,9 @@ Pod::Spec.new do |s|
   }
 
   s.cocoapods_version = '>= 1.4.0'
-  s.swift_version = '4.2'
+  if s.respond_to?(:swift_versions) then
```

Comment:
I'd remove the condition and bump `s.cocoapods_version` to `>= 1.7.0` when we release Nimble 9.